db/migrate/20170919013043_create_subtasks.rb

```diff
@@ -2,7 +2,7 @@ class CreateSubtasks < ActiveRecord::Migration[5.1]
   def change
     create_table :subtasks do |t|
       t.string :title
-      t.time :time
+      t.integer :time
```

Comment:
you might avoid chnge old migration file.
that make `db/scheama.rb` different from real database scheama.

however this rails application doesn't have production enviroment, you execute `rails db:migrate:reset` if you have this problem.